submissions/korotushak96/a-tiny-JS-world/index.js

```diff
@@ -73,8 +73,12 @@ const catWoman = {
     saying: cat.saying
 };
 
+let getParameters = ({species, name, gender, legs, hands, saying}) =>{
```

Comment:
Please, use `const` for define functions.

Reply:
I add a few, but I don't undrestand reasons in this case. Please, explaine.

Reply:
okay ;)